CHANGELOG.md

```diff
@@ -16,8 +16,10 @@ This project does NOT adhere to [Semantic Versioning](https://semver.org/spec/v2
 * `[sitecore-jss-angular][templates/angular]` jss-angular package and sample has been updated to version 14. This means several changes:
   * JSS Angular sample is now using Ivy
   * IE11 no longer supported by JSS Angular
+  * You can no longer specify lazy-loaded routes by setting a string value to loadChildren. Make sure you move to dynamic ESM import statements. For example, `loadChildren: () => import('./modules/my.module').then(m => m.MyModule)` should be used instead of `loadChildren: './modules/my.module'`. You can find more info on dunamic imports here:
```

Comment:
We've already migrated to new format, it was a breaking change in Angular 8 as I remember: https://github.com/Sitecore/jss/blob/dev/CHANGELOG.md?plain=1#L586
In this case, what is changed now?

Reply:
Basically during upgrade to v14 you added deprecated `string` support and now you revert it back